ruby/lib/minitest/queue/build_status_recorder.rb

```diff
@@ -52,6 +52,7 @@ def record(test)
         if (test.failure || test.error?) && !test.skipped?
           build.record_error("#{test.klass}##{test.name}", dump(test), stats: stats)
         else
+          build.record_flaky("#{test.klass}##{test.name}", stats: stats)
```

Comment:
Also might make sense to add this to `record_success` pipeline so we don't need to do two roundtrips.